### PR TITLE
feat: add region and shortcode label

### PR DESCRIPTION
**Pull request by jim-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   ...

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/appstore/src/components/cfds-listing/index.tsx [hunk 1]
```diff
@@ -35,12 +35,21 @@ const CFDsListing = () => {
...
 
-    const getShortCode = (account: TDetailsOfEachMT5Loginid) => {
-        return account.landing_company_short &&
-            account.landing_company_short !== 'svg' &&
-            account.landing_company_short !== 'bvi'
-            ? account.landing_company_short?.charAt(0).toUpperCase() + account.landing_company_short?.slice(1)
-            : account.landing_company_short?.toUpperCase();
+    const getShortCodeAndRegion = (account: TDetailsOfEachMT5Loginid) => {
+        let short_code_and_region;
+        if (!is_demo && !is_eu_user) {
```
Comment (matin-binary) [inline packages/appstore/src/components/cfds-listing/index.tsx:40]: no so readable, e.g. `!is_demo` should be `is_real`

File: packages/appstore/src/components/containers/trading-app-card.tsx [hunk 2]
```diff
@@ -34,9 +34,14 @@ const TradingAppCard = ({
...
             <div className={classNames('trading-app-card__details', { 'trading-app-card--divider': has_divider })}>
-                <Text className='title' size='xs' line_height='s'>
-                    {sub_title}
-                </Text>
+                <div>
+                    <Text className='title' size='xs' line_height='s'>
+                        {sub_title}
+                    </Text>
+                    {short_code_and_region && (
+                        <span className='trading-app-card__details__short-code'>{short_code_and_region}</span>
```
Comment (matin-binary) [inline packages/appstore/src/components/containers/trading-app-card.tsx:42]: Change this to `<Text>` component and use the props for that component such as size and weight and ...

File: packages/appstore/src/components/cfds-listing/index.tsx [hunk 3]
```diff
@@ -35,12 +35,21 @@ const CFDsListing = () => {
...
+            short_code_and_region =
+                account.landing_company_short &&
+                account.landing_company_short !== 'svg' &&
+                account.landing_company_short !== 'bvi'
+                    ? account.landing_company_short?.charAt(0).toUpperCase() + account.landing_company_short?.slice(1)
+                    : account.landing_company_short?.toUpperCase();
+            short_code_and_region =
+                account.market_type !== 'financial' && account.landing_company_short !== 'bvi'
+                    ? `${short_code_and_region} - ${account?.server_info?.geolocation?.region}`
+                    : short_code_and_region;
```
Comment (matin-binary) [inline packages/appstore/src/components/cfds-listing/index.tsx:50]: Why are we defining short_code_and_region 2 times?

Comment (jim-deriv) [reply packages/appstore/src/components/cfds-listing/index.tsx:50]: @matin-binary, I'm not redefining it, i'm reassigning it a value based on another check to add the region to it at the end

File: packages/appstore/src/components/cfds-listing/index.tsx (hunk 1 above)
Comment (jim-deriv) [reply packages/appstore/src/components/cfds-listing/index.tsx:40]: Sure @matin-binary, I'll add another  computed to the store for `is_real
`

File: packages/appstore/src/components/cfds-listing/index.tsx (hunk 3 above)
Comment (matin-binary) [reply packages/appstore/src/components/cfds-listing/index.tsx:50]: in that case we need to have 2 separate variables.

**Review by matin-binary — APPROVED** [review]

**Pull request merged**